spec/fixtures/pulpcore-migration/katello-answers-after.yaml

```diff
@@ -22,6 +22,7 @@ foreman::plugin::dhcp_browser: false
 foreman::plugin::expire_hosts: false
 foreman::plugin::git_templates: false
 foreman::plugin::google: false
+foreman::plugin::hdm: false
```

Comment:
these changes need to happen also to the other existing `pulpcore-*` fixtures, then the tests will pass

Reply:
I've been debating if we need to drop the test - at this point it's not giving us anything and only causing overhead.